### PR TITLE
Use the simple-phpunit script from the Symfony PHPUnit bridge

### PR DESCRIPTION
**Pull request by leofeyer**
As briefly discussed with @Toflar in Slack. 😉

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -100,17 +100,6 @@ jobs:
 before_install:
     - |
       # Helper functions
-      run_tests () {
```
Comment (Toflar) [inline .travis.yml:103]: Could you maybe describe what you did here? This doesn't look like just switching to simple phpunit but a lot more :)

File: phpunit.xml.dist [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            <directory>./faq-bundle/src</directory>
+            <directory>./manager-bundle/src</directory>
+            <directory>./news-bundle/src</directory>
+            <exclude>
+                <directory>./calendar-bundle/src/Resources</directory>
+                <directory>./core-bundle/src/Resources</directory>
+                <directory>./faq-bundle/src/Resources</directory>
+                <directory>./manager-bundle/src/Resources</directory>
+                <directory>./news-bundle/src/Resources</directory>
+                <!-- exclude files with symbols and side-effects -->
```
Comment (Toflar) [inline phpunit.xml.dist:49]: What are does?

File: .travis.yml (hunk 1 above)
Comment (leofeyer) [reply .travis.yml:103]: Now that there is a global `phpunit.xml.dist` file, we do not need the `run_tests` helper method anymore.

File: phpunit.xml.dist (hunk 2 above)
Comment (leofeyer) [reply phpunit.xml.dist:49]: I have just copied this from the existing `core-bundle/phpunit.xml.dist` file:

https://github.com/contao/contao/blob/198adf77b8d3d50dd8a664062814bdce216d9025/core-bundle/phpunit.xml.dist#L42-L45

It excludes classes with symbols and side-effects from the coverage report.

File: .travis.yml (hunk 1 above)
Comment (Toflar) [reply .travis.yml:103]: Okay, but running only the tests for one bundle would still be useful during development.

Comment (leofeyer) [reply .travis.yml:103]: You still can.

```
vendor/bin/phpunit
# runs all unit tests

vendor/bin/phpunit -c calendar-bundle
# runs only the tests in calendar-bundle/tests
```

Comment (Toflar) [reply .travis.yml:103]: Ah man, I've confused this with your run shell script 🤦‍♂

**Review by Toflar — APPROVED** [review]

**Review by bytehead — APPROVED** [review]
LGTM

File: .travis.yml (hunk 1 above)
Comment (leofeyer) [reply .travis.yml:103]: np

**Pull request merged**